cirq/experiments/cross_entropy_benchmarking.py

```diff
@@ -25,6 +25,52 @@
 
 CrossEntropyPair = NamedTuple('CrossEntropyPair', [('num_cycle', int),
                                                    ('xeb_fidelity', float)])
+SpecklePurityPair = NamedTuple('SpecklePurityPair', [('num_cycle', int),
+                                                     ('purity', float)])
+
+
+def purity_from_probabilities(
+        hilbert_space_dimension: int,
+        probabilities: Sequence,
```

Comment:
`Sequence[float]`

Reply:
Done. I was originally using this on a list of numpy arrays, but I've modified that code to convert to a flattened numpy array to match Sequence[float].